suppressions.xml

```diff
@@ -39,4 +39,12 @@
         <packageUrl regex="true">^pkg:maven/org\.glassfish/javax\.json@.*$</packageUrl>
         <vulnerabilityName>CVE-2023-7272</vulnerabilityName>
     </suppress>
+    <suppress until="2025-12-20">
```

Comment:
How long do we usually suppress these for?

Reply:
not sure about the rewrite side, but I made it a little shorter.